lib/graphql/relay/connection/ecto.ex

```diff
@@ -63,16 +63,17 @@ if Code.ensure_loaded?(Ecto) do
       a_after = cursor_to_offset(args[:after])
       first = args[:first]
       last = args[:last]
+      where_property = args[:where] || :id
```

Comment:
I like `ordered_by` as well. Think that's a little more intuitive. Pushed a fix.